pkg/model/config/config.go

```diff
@@ -143,34 +158,62 @@ func (c Config) resourceAPIVersionCompatible(verType, version string) bool {
 	return true
 }
 
-// GVK contains information about scaffolded resources
-type GVK struct {
+// Resource contains information about scaffolded resources
+type Resource struct {
 	Group   string `json:"group,omitempty"`
 	Version string `json:"version,omitempty"`
 	Kind    string `json:"kind,omitempty"`
 
-	// CRDVersion holds the CustomResourceDefinition API version used for the GVK.
+	// CRDVersion holds the CustomResourceDefinition API version used for the Resource.
 	CRDVersion string `json:"crdVersion,omitempty"`
-	// WebhookVersion holds the {Validating,Mutating}WebhookConfiguration API version used for the GVK.
+	// WebhookVersion holds the {Validating,Mutating}WebhookConfiguration API version used for the Resource.
 	WebhookVersion string `json:"webhookVersion,omitempty"`
+
+	// Webhooks holds the Webhooks data
+	Webhooks Webhook `json:"webhooks,omitempty"`
 }
 
-// isEqualTo compares it with another resource
-func (r GVK) isEqualTo(other GVK) bool {
+// Webhook contains information about scaffolded webhooks
+type Webhook struct {
+	Defaulting bool `json:"defaulting,omitempty"`
+	Validation bool `json:"validation,omitempty"`
+	Conversion bool `json:"conversion,omitempty"`
```

Comment:
These will be unmarshaled weirdly if not set, ex `webhooks: {}`. I think this should work
```suggestion
	Defaulting bool `json:"defaulting"`
	Validation bool `json:"validation"`
	Conversion bool `json:"conversion"`
```

Reply:
`omitempty` means that the attributes can be empty then `webhooks: {}` means that it is empty which matches with how its rules. I mean, I resource can have 0 to Many webhooks. 

However, if we follow up your suggestion it is ok but we will store all attributes with the false value always when none webhooks were scaffold which is unnecessary . E.g 

```yaml
webhooks:
    conversion: false
    defaulting: false
    validation: false

```

So, I think that keep `omitempty` (allows be empty) still better.

Reply:
I think I'm ok with this.